package.json

```diff
@@ -1,124 +1,115 @@
+    "test:prod": "npm run start-rpc -- \"jest --no-cache --env=./test/env.js\"",
+    "test:watch:windows": "concurrently \"npm run start-rpc\" \"jest --coverage --env=./test/env.js --watchAll\"",
+    "change_resolver": "ts-node -O '{\"module\": \"commonjs\"}' ./scripts/change-resolver.ts"
+  },
+  "keywords": [
+    "decentralized",
+    "identity",
+    "access",
+    "library",
+    "energy web",
+    "iam",
+    "switchboard"
+  ],
+  "author": "Kim Honoridez <kim.energyweb@gmail.com> (https://github.com/kim-energyweb)",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/energywebfoundation/iam-client-lib.git"
+  },
+  "homepage": "https://github.com/energywebfoundation/iam-client-lib.git#readme",
+  "license": "GPL-3.0-or-later",
+  "private": false,
+  "engines": {
+    "node": ">= 10.0.0",
+    "npm": ">= 6.0.0"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.12.5",
+    "@energyweb/iam-contracts": "1.17.2",
+    "@ensdomains/ens": "^0.4.5",
+    "@ew-did-registry/claims": "0.5.2-alpha.1076.0",
+    "@ew-did-registry/did": "0.5.2-alpha.1076.0",
+    "@ew-did-registry/did-document": "0.5.2-alpha.1076.0",
+    "@ew-did-registry/did-ethr-resolver": "0.5.2-alpha.1076.0",
+    "@ew-did-registry/did-resolver-interface": "0.5.2-alpha.1076.0",
+    "@ew-did-registry/jwt": "0.5.2-alpha.1076.0",
+    "@ew-did-registry/keys": "0.5.2-alpha.1076.0",
+    "@ew-did-registry/proxyidentity": "0.5.2-alpha.1076.0",
+    "@gnosis.pm/safe-apps-sdk": "1.0.3",
+    "@metamask/detect-provider": "^1.2.0",
+    "@walletconnect/web3-provider": "1.0.0-rc.4",
+    "axios": "^0.21.1",
+    "base64url": "^3.0.1",
+    "concurrently": "^6.1.0",
+    "eth-ens-namehash": "^2.0.8",
+    "ethers": "4.0.45",
+    "js-sha3": "^0.8.0",
+    "lodash.difference": "^4.5.0",
+    "nats.ws": "1.0.0-111",
+    "qs": "^6.9.4",
+    "tslib": "^2.0.3",
+    "uuid": "^7.0.3"
+  },
+  "devDependencies": {
+    "@babel/core": "7.12.10",
+    "@babel/plugin-transform-regenerator": "7.12.1",
+    "@babel/preset-env": "7.12.11",
+    "@energyweb/km-crypto": "file:energyweb-km-crypto-v1.0.0.tgz",
+    "@energyweb/km-utils": "file:energyweb-km-utils-v1.0.0.tgz",
+    "@typechain/ethers-v4": "^1.0.1",
+    "@types/jest": "^26.0.13",
+    "@types/lodash.difference": "^4.5.6",
+    "@types/qs": "^6.9.5",
+    "@types/uuid": "^8.3.0",
+    "@typescript-eslint/eslint-plugin": "^4.0.1",
+    "@typescript-eslint/parser": "^4.0.1",
+    "babel-jest": "26.6.3",
+    "bili": "^5.0.5",
+    "copyfiles": "^2.4.0",
+    "coveralls": "^3.1.0",
+    "eslint": "^7.8.1",
+    "ganache-cli": "^6.12.2",
+    "husky": "^4.3.8",
+    "jest": "^26.4.2",
+    "jest-config": "^26.4.2",
+    "jest-environment-jsdom": "^26.6.2",
+    "rimraf": "^3.0.2",
+    "rollup-plugin-typescript2": "^0.27.2",
+    "run-with-testrpc": "0.3.1",
+    "semantic-release": "17.3.0",
+    "ts-jest": "26.3.0",
+    "typechain": "^2.0.0",
+    "typedoc": "^0.21.5",
+    "typedoc-plugin-markdown": "~3.10.4",
```

Comment:
not sure why it shows that whole file is updated, I only updated `typedoc` and `typedoc-plugin-markdown` module versions

Reply:
I tried npm I and it does not modify the package.json, seems Like I did git stash and then git stash pop, thats can be a reason, currently its fine after npm i

Reply:
If this changes are unintentional, could you please revert them?

Reply:
seems Like because of that stash changes package.json was reverted back to older version, I autoformatted the code and added couple missing dependencies(eslint-prettier plugins) and added also lint-staged https://github.com/energywebfoundation/iam-client-lib/pull/226/commits/f82c8b07e9c7e10a50e7e6d9d35146cf367dec96

please check